include/psa/crypto_config.h

```diff
@@ -1574,8 +1574,8 @@
  *
  * \note  This feature is incompatible with insecure block cipher,
  *        MBEDTLS_DES_C, and cipher modes which always require decryption
- *        operation, MBEDTLS_CIPHER_MODE_CBC, MBEDTLS_CIPHER_MODE_XTS and
- *        MBEDTLS_NIST_KW_C. This feature is incompatible with following
```

Comment:
I'm not sure I agree with this removal. `NIST_KW_C` uses `ALG_ECB_NO_PADDING` which is still listed as incompatible with this feature.